paradox-nakadi-consumer-example-boot/README.md

```diff
@@ -29,4 +29,4 @@
     java -Dspring.profiles.active=local-zk -jar ./paradox-nakadi-consumer-example-boot-0.0.1-SNAPSHOT.jar
 
 
-
+For the mock server, there are two sample responses under resources folder.
```

Comment:
I don't understand this sentence. *Mock server* is not mentioned anywhere else in the `README.md`